apps/web/app/(app)/[emailAccountId]/assistant/RuleForm.tsx

```diff
@@ -849,7 +860,7 @@ export function RuleForm({
           )}
 
           {rule.id ? (
-            <Button type="submit" loading={isSubmitting}>
+            <Button type="submit" loading={isSubmitting} disabled={isDeleting}>
```

Comment:
`loading={isSubmitting || isDeleting}` would work. and show loading. it will auto set disabled

Reply:
You don't need disabled if loading is set (if you look in the Button code)

Reply:
ah my bad. it's a different button